package fedoseev.jobboard.dto.request;

import jakarta.validation.constraints.NotBlank;
import jakarta.validation.constraints.Size;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor

public class CompanyRequest {

    @NotBlank
    @Size(max = 255)
    private String name;

    private String description;

    private String logoUrl;

    private String website;



}
